app/assets/stylesheets/Bars.css.sass

```diff
@@ -0,0 +1,43 @@
+.navbar-brand
```

Comment:
You don't need to have `.css.sass`. Just `.sass` is fine. Also usually always a good idea to lowercase filenames.